src/index.ts

```diff
@@ -173,39 +186,48 @@ export interface CliArguments {
   flow: boolean;
   update: boolean;
   init: boolean;
+  ignoreUntracked: boolean;
 }
 
-yargs
-  .scriptName('unimported')
-  .usage('$0 <cmd> [args]')
-  .command(
-    '*',
-    'scan your project for dead files',
-    (yargs) => {
-      yargs.option('init', {
-        alias: 'i',
-        type: 'boolean',
-        describe: 'dump default settings to .unimportedrc.json',
-      });
-
-      yargs.option('flow', {
-        alias: 'f',
-        type: 'boolean',
-        describe: 'indicates if your code is annotated with flow types',
-      });
-
-      yargs.option('update', {
-        alias: 'u',
-        type: 'boolean',
-        describe: 'update the ignore-lists stored in .unimportedrc.json',
-      });
-    },
-    function (argv: Arguments<CliArguments>) {
-      return main({
-        init: argv.init,
-        update: argv.update,
-        flow: argv.flow,
-      });
-    },
-  )
-  .help().argv;
+if (process.env.NODE_ENV !== 'test') {
```

Comment:
There was a race condition in the tests with this running in the background. The addition of the additional async calls for the git logic, even though mocked, exposed this, leading to inconsistent behavior within the jest tests.